crates/lib/Makefile.toml

```diff
@@ -21,6 +21,10 @@ run_task = [{name = "test-flow"}]
 [tasks.lint]
 dependencies = ["clippy-flow"]
 
+[tasks.examples]
+command = "cargo"
+args = ["build", "--examples", "--features", "libquil"]
```

Comment:
Is `--features libquil` required given the change on line 3?

Reply:
I think so. Those are the features passed to `cargo test`.